WooCommerce/WooCommerce.xcodeproj/project.pbxproj

```diff
@@ -11036,7 +11032,7 @@
 				ASSETCATALOG_COMPILER_WIDGET_BACKGROUND_COLOR_NAME = WidgetBackground;
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++17";
 				CLANG_ENABLE_OBJC_WEAK = YES;
-				CODE_SIGN_ENTITLEMENTS = StoreWidgets/Entitlements/StoreWidgets-Debug.entitlements;
+				CODE_SIGN_ENTITLEMENTS = "StoreWidgets/Entitlements/StoreWidgets-Debug.entitlements";
```

Comment:
I'm not sure how this change appeared here, possibly some branch switcheroo mistake went into the PR while testing. I'll be sure to remove any changes with the review, and before merging.